app/packs/src/apps/mydb/elements/details/VersionsTable.js

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable react/forbid-prop-types */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Pager } from 'react-bootstrap';
+import BootstrapTable from 'react-bootstrap-table-next';
+import VersionsFetcher from 'src/fetchers/VersionsFetcher';
+import VersionsTableTime from 'src/apps/mydb/elements/details/VersionsTableTime';
+import VersionsTableChanges from 'src/apps/mydb/elements/details/VersionsTableChanges';
+import { elementShowOrNew } from 'src/utilities/routesUtils';
+import DetailActions from 'src/stores/alt/actions/DetailActions';
+import SamplesFetcher from 'src/fetchers/SamplesFetcher';
+import ResearchPlansFetcher from 'src/fetchers/ResearchPlansFetcher';
+import ScreensFetcher from 'src/fetchers/ScreensFetcher';
+import WellplatesFetcher from 'src/fetchers/WellplatesFetcher';
+
+export default class VersionsTable extends Component {
```

Comment:
Maybe convert this to a functional component? I feel like introducing new components might be an opportunity to move the codebase away from class components.